prototype/jurisdictions/certificate.AU.xml

```diff
@@ -27,23 +26,12 @@
 					<option value="unsure">
 						<label>you're not sure if you have the rights to publish this data as open data</label>
 					</option>
-					<option value="complicated">
```

Comment:
Another example is the Land Registry Price Paid data in the UK, which is mostly open but contains some Ordnance Survey and Royal Mail rights that aren't clear.